pymc3/model.py

```diff
@@ -434,6 +439,23 @@ def __init__(self, name='', model=None):
             self.potentials = treelist()
             self.missing_values = treelist()
 
+        if theano_config is None:
```

Comment:
You mean adding the testval option even when a config dict is specified? I guess we could add it if it is not already in there, I'm not sure about this though.